src/test/integration/bug-39-parse-error-it/validate.groovy

```diff
@@ -1,3 +1,5 @@
+import org.apache.commons.lang3.SystemUtils
```

Comment:
My (personal) preference: reduce the list of dependencies (commons-lang) in this case and use plexus-utils which is part of the build anyway.

Commons and guava are not needed anymore after switching to java 8.

https://github.com/sonatype/plexus-utils/blob/master/src/main/java/org/codehaus/plexus/util/Os.java

`if (Os.isFamily(FAMILY_WINDOWS)) { … }`

Reply:
Great! Thanks for your hint.